web/src/components/feature.js

```diff
@@ -60,6 +58,7 @@ export default function Feature({ content, image }) {
 }
 
 Feature.propTypes = {
+    bgColor: PropTypes.string.isRequired,
```

Comment:
Does making this a required prop break it's usage elsewhere?

Reply:
no, but it does break if you remove it

Reply:
So what happens when we use it elsewhere and don't specify a `bgColor` prop?

Reply:
@asasmith we arent using this right now. This is from when I tried to use this instead of creating a new component

Reply:
If we aren't using it anywhere else is there a need to make the background color flexible?

Reply:
Looks like this feature (bgColor) should be used on a few components: both of the about pages and the contact pages. https://www.figma.com/file/y6cqFJnepTl3UKvMMM22jE/Hubble-Technology?node-id=460%3A924